app/components/Button.tsx

```diff
@@ -27,6 +28,10 @@ export interface ButtonProps extends PressableProps {
    * The text to display if not using `tx` or nested components.
    */
   text?: TextProps["text"]
+  /**
+   * Optional props to pass to the Text
+   */
+  textProps?: RNTextProps
```

Comment:
Small nitpick here, we should probably proper case it here for consistency across the other Ignite components, `TextProps`

![image](https://user-images.githubusercontent.com/374022/208725777-d19e55fb-2df5-4f20-85b5-fb70a88565c8.png)